packages/mc-scripts/src/cli.ts

```diff
@@ -197,6 +198,43 @@ async function run() {
       }
     );
 
+  // Command: deployment-previews:push
+  const usageDeploymentPreviewsPush =
+    'Creates or updates a deployment preview for the customization application.';
```

Comment:
I was intentionally general because I think we need to to support deployment previews in Custom Views as well.

That's something we had in that's feature plan but left out of the first iteration.

Reply:
Let's have a separate discussion about this as I'm not sure it's something we can easily support.

Reply:
Message updated here: e624165c7d4d0aaa07e2a0e0c8004d380171d750

Reply:
This one hasn't been updated?

Reply:
Sorry about it.

Updated here: adf3d479abcce1134a8b693d89ba1602c1c3ecaf